clients/lsp-eslint.el

```diff
@@ -233,8 +233,10 @@ source.fixAll code action."
   (lsp-stdio-connection
    (lambda () lsp-eslint-server-command)
    (lambda ()
-     (and (cl-second lsp-eslint-server-command)
-          (file-exists-p (cl-second lsp-eslint-server-command)))))
+     (or
+      (not (string-equal (cl-first lsp-eslint-server-command) "node"))
```

Comment:
can you use `(f-base (f-filename (cl-first lsp-eslint-server-command)))`  to handle the case when the user has provided full path to node? Also, maybe string-equal has to be changed to contains? IDK how common is to have node binary in format node12.exe or something like that.

Reply:
@yyoncho updated!